NEWS.md

```diff
@@ -1,3 +1,12 @@
+## 26.0.0 IN-PROGRESS
```

Comment:
The version in the pom.xml should also be updated to match up with this

Reply:
It's updated
![image](https://user-images.githubusercontent.com/5904625/213165425-a9e30a77-0030-4241-bd97-774025c74b0c.png)